CMakeLists.txt

```diff
@@ -1,30 +1,15 @@
-cmake_minimum_required(VERSION 3.29)
+cmake_minimum_required(VERSION 3.25)
```

Comment:
Why the downgrade? Totally fine, just curious.

Reply:
Was a mistake. Unintentional. I will revert!